### DISCUSSION TITLE
Adding photos to POS app not possible via front-end

### DISCUSSION THREAD

**Discussion by c3piio** [btcpayserver/btcpayserver#4860] [Technical Support] [answered]
Hello,

Per the [documentation for the POS app](https://docs.btcpayserver.org/Apps/#point-of-sale-app) it should be possible to add a picture to the products when adding them. However, the only option given is providing a url, such as `~/img/pos-sample/green-tea.jpg`

The problem is, I cannot locate /img/pos-sample/ on the luna node deployment. From the documentation it seems like images can be uploaded to the products (like is possible for other parts of the server like branding) but the products only allow a url.

1. Where can the /img/ folder be found on the server?
2. Is there another way to easily add product photos? /img/ (if it is indeed in /var/lib/docker/volumes/generated_btcpayserver_data/_data/PointOfSale/) seems to require SU which makes access via SFTP complicated, since my client doesn't allow me to run commands.

**Comment by pavlenex** [comment]
You can use any external site and juts link it, or if you're a server admin, then you can upload the file directly from your server.

**Reply by c3piio** [reply]
I am a server admin, but I wasn't able to locate the /img/ folder the POS app relies on. Are you able to point me in the right direction?

**Reply by dennisreimann** [reply, marked as answer]
You can upload files and copy their links in the Server Settings  > Files section.
